common/config/eventlog-config-hook.yml

```diff
@@ -0,0 +1,19 @@
+---
+defaults:
+  http_success_codes:
+  - 200
+  - 201
+  http_inprogress_codes:
+  - 202
+  log_inprogress_state: true # flag that is used to determine if async operations in-progress state is to be logged
+  cf_last_operation: false # flag to indicate if a particular HTTP operation is to be treated as cf-lastoperation semantics additionally
+  ignore_service_fabrik_operation: false # flag to indicate if service-fabrik operations are to be excluded. for ex. broker update (HTTP Patch)
+                                         #  gets invoked via backup/restore etc, which are to be ignored when coming via HTTP Patch.
+  include_response_body: false # wheter HTTP response body is to be logged after sanitization or not
+"/hook":
+  POST:
+    enabled: true
+    event_name: get_action_response
```

Comment:
Yes. I have disabled log_event config.
https://github.com/cloudfoundry-incubator/service-fabrik-boshrelease/pull/77/files#diff-84069eb965565d69fcfb2ab09719da23R68
Had included this file, in case we require it event log future.

Reply:
Removed as discussed